internal/cmd/cmd_getdel.go

```diff
@@ -0,0 +1,92 @@
+package cmd
+
+import (
+	"github.com/dicedb/dice/internal/object"
+	dstore "github.com/dicedb/dice/internal/store"
+	"github.com/dicedb/dicedb-go/wire"
+)
+
+var cGETDEL = &DiceDBCommand{
+	Name:      "GETDEL",
+	HelpShort: "GETDEL returns the value for the queried key in args.",
```

Comment:
The short documentation seems wrong here. it should be - `GETDEL returns the value of the key and then deletes the key.`